beginner_source/translation_transformer.py

```diff
@@ -0,0 +1,399 @@
+"""
+Language Translation with Transformer
+=====================================
+**Author**: `Jayasimha T <https://github.com/iamsimha>`_
+
+This tutorial shows, how to train a translation model from scratch using
+Transformer. We will be using `Multi30k
+dataset <https://github.com/multi30k/dataset>`__ to train a German to
+English translation model.
+
+The data processing and data loading code is taken from an `earlier version
```

Comment:
I don't think we need to add a reference to the old tutorial. It's like giving credits to ourself. :)